### DISCUSSION TITLE
Bitte Dienste-Kontrolle für 2 Versionen: 7490/Labor latest + 7590 final 7.50

### DISCUSSION THREAD

**Discussion by maverick3005** [Freetz-NG/freetz-ng#757] [Allgemein]
Folgende Sachen sind mir im Freetz WebIf aufgefallen:

Zur 7490 latest LabBeta:

![7490](https://user-images.githubusercontent.com/119104833/222938802-ceb38c75-cc23-4046-82c8-e595f5d94e3f.png)

Hier: pcpd FAILED (also Portforwarding-Daemon) und capitcp FAILED

Bei der 7590 7.50 FINAL habe ich folgendes beobachtet:

![7590](https://user-images.githubusercontent.com/119104833/222938830-2e0ea372-b676-456c-b463-27dacbc58c9f.png)

Hier: meshd_config FAILED

bei der 6890 + 5590 (beide LabBeta 7.51x) hingegen habe ich hier ACTIVE oder INACTIVE sauber stehen.

Alle Test-Versionen sind identisch gebaut, da wurde keiner der Dienste entfernt. Einzig "MyFritz" und "tr069" sind überall draussen,
ferner noch die "Leftovers" sind mit draussen.

Ich kann im Moment keine Mesh-Config, oder Portforwarding testen, daher bitte ich die Community bei der 7490+7590
hier um Überprüfung ob Portforwarding bei der 7490 noch rennt (PCP-Sessions) und bei der 7590 wie es mit der Mesh-Config
in einem laufenden System gerade aussieht! capitcp wird meines Wissens noch für CAPI-Fax und evtl. Apps verwendet,
die im lokalen Netz die Box steuern können ?

Evtl. ist noch eine Anpassung im "rc-start/svctl"-Handling erforderlich, oder es handelt sich lediglich beim Freetz WebIF um einen
Anzeigefehler ?

Möglich wäre, das diese Dienste auch erst dann "aktiv" werden, wenn sie im AVM-Webinterface "aktiv" erst in Verwendung sind,
und das Handling dann auf den verschiedenen Modellen unterschiedlich gehandhabt wird ? - Das erscheint mir aber zumindest
bei der Mesh-config unlogisch da dieser Dienst auf allen Geräten immer aktiv ist. Möglich das aufgrund von Speichermangel dieser Dienst
hier erst dann gestartet wird, wenn die Dienste "aktiv" auch benötigt werden und im Anschluss wieder beendet werden ?

Wenn der capitcp-Dienst auf der 7490 nicht aktiv ist würde mir das auch logisch erscheinen, wenn er im AVM-Webif nicht aktiviert wurde, ebenfalls aus Gründen hier Speicher zu sparen ?

Dann sollte jedoch das Freetz-WebIF in besten Falle "orange" den Dienst als "inactive" anzeigen normalerweise und nicht als "failed".

**Comment by fda77** [comment]
Es wird einfach angezeigt was avm ausgibt. Für den Fall dass noch mehr damit nicht klar kommen müsste ich die Übersicht wieder entfernen.
Wenn du der Meinung bist irgend ein "service" sollte einen anderen status haben, untersuch weshlab er in diesem status ist. Ich mach das nicht. Am besten zuerst mal anscheuen was zb "inactive" bedeutet

**Comment by maverick3005** [comment]
Deshalb ging in diesem Fall ja die "funktionelle" Frage an die Community und Nutzer genau dieser Dienste ;-)

**Comment by fda77** [comment]
Bei mir sind alle etweder active oder inactive. Ich hab aber viel von dem AVM-Krempel entfernt und das Gerät ist nur dsl-MODEM

**Reply by maverick3005** [reply]
Korrekt, als reines Modem problemlos. "fullbridge" ON und Ruhe ;-)
